README.md

```diff
@@ -30,7 +30,7 @@ Azure service crates generated from [Azure REST API Specifications](https://gith
 
 ## Status
 
-🚨 WARNING 🚨: This project is under active development. Be aware that large breaking changes will happen before 1.0 is reached.
+🚨 WARNING 🚨: This project is no longer under active development. Work on officially supported crates is being done in the `main` branch. Pull requests to `legacy` may be considered but are not guaranteed to be merged. Instead, you might consider forking this repository, creating a topic branch you'd need to submit a pull request anyway, and [override dependencies](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) in your `Cargo.toml` file(s) as necessary.
```

Comment:
I don't like suggesting they fork. I'd rather they migrate to the new crates

Reply:
That text is already above. I'm taking into account - much further below in this file - that they may not be *able* to, especially before they are ready. We could opt not to suggest something (and I intentionally do not go into too many details), but that may just lead to more issues being opened. If that's the case, we should consider a bot response.

Reply:
Can we say please migrate to the new crates or open issues/PRs against main. Forking as a last resort .....

// @ronniegeraghty Any opinions on this wording?

Reply:
> That text is already above. I'm taking into account - much further below in this file - that they may not be _able_ to, especially before they are ready. We could opt not to suggest something (and I intentionally do not go into too many details), but that may just lead to more issues being opened. If that's the case, we should consider a bot response.

Makes sense. I'm hoping to get issues so we learn who was using these crates and which ones we might want to prioritize

Reply:
Maybe adding something like: "These crates should NEVER be used in a production setting."

Reply:
Too late. But at least for 1P, the "unsupported" should've implied that.